src/adhocracy_mercator/adhocracy_mercator/scripts/sample_badge.json

```diff
@@ -0,0 +1,24 @@
+[
```

Comment:
normally we do test driven development and 100% unit/integration test coverage for all backend code.
I would be good if we also start doing this for mercator scripts instead manual testing with sample scripts. An example is here: 
https://github.com/liqd/adhocracy3.mercator/blob/master/src/adhocracy_core/adhocracy_core/scripts/test_import_users.py

If setting up an intergration test is to trouble some we could also make it a functional test